### PR TITLE
Querystring plugin uses lates SDK and new utils package versions

### PR DESCRIPTION
**Pull request by deepakvk**
Upgraded the SDK dependency to use version 1.0.0 and added the new devdependency @axway/api-builder-test-utils@1.0.0 and changed where mockRuntime is required from in the unit tests.

### CODE REVIEW

File: flow-nodes/api-builder-plugin-querystring/package.json [hunk 1]
```diff
@@ -20,14 +20,15 @@
 		"url": "https://github.com/axway/api-builder-examples"
 	},
 	"dependencies": {
-		"@axway/api-builder-sdk": "^0.2.0"
+		"@axway/api-builder-sdk": "^0.8.0"
```
Comment (pltod2) [inline flow-nodes/api-builder-plugin-querystring/package.json:23]: change this to ^1.0.0 when we release

File: flow-nodes/api-builder-plugin-querystring/package.json [hunk 2]
```diff
@@ -20,14 +20,15 @@
...
 	},
 	"peerDependencies": {
 		"@axway/api-builder-runtime": "^4.5.0"
 	},
 	"devDependencies": {
 		"chai": "^4.1.2",
-		"mocha": "^6.1.4"
+		"mocha": "^6.1.4",
+		"@axway/api-builder-test-utils": "0.1.0"
```
Comment (pltod2) [inline flow-nodes/api-builder-plugin-querystring/package.json:31]: change this to ^1.0.0 when we release

File: flow-nodes/api-builder-plugin-querystring/src/index.js [hunk 3]
```diff
@@ -4,10 +4,16 @@ const actions = require('./actions');
...
+ *   from API Builder config.pluginConfig['api-builder-plugin-pluginName']
+ * @param {string} pluginConfig.proxy - The configured API-builder proxy server
+ * @param {object} options - Additional options and configuration provided by API Builder
+ * @param {string} options.appDir - The current directory of the service using the plugin
+ * @param {string} options.logger - An API Builder logger scoped for this plugin
  * @returns {object} An API Builder plugin.
  */
-async function getPlugin() {
-	const sdk = new SDK();
+async function getPlugin(pluginConfig, options) {
```
Comment (pltod2) [inline flow-nodes/api-builder-plugin-querystring/src/index.js:15]: options is not used ... remove

Comment (deepakvk) [reply flow-nodes/api-builder-plugin-querystring/src/index.js]: Removed it.

File: flow-nodes/api-builder-plugin-querystring/package.json (hunk 2 above)
Comment (deepakvk) [reply flow-nodes/api-builder-plugin-querystring/package.json]: Bumped the version.

File: flow-nodes/api-builder-plugin-querystring/package.json (hunk 1 above)
Comment (deepakvk) [reply flow-nodes/api-builder-plugin-querystring/package.json]: Bumped the version

File: flow-nodes/api-builder-plugin-querystring/.gitignore [hunk 4]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+package-lock.json
```
Comment (wickedest) [inline flow-nodes/api-builder-plugin-querystring/.gitignore:2]: newline

File: flow-nodes/api-builder-plugin-querystring/src/flow-nodes.yml [hunk 5]
```diff
@@ -5,123 +5,135 @@ flow-nodes:
...
+          context: $.decodedStr
+          schema:
+            type: string
+         # "Error" (error) output
+        throws:
+          name: Error
+          description: An unexpected error was encountered.
+          context: $.error
+          schema:
+            type: string
```
Comment (wickedest) [inline flow-nodes/api-builder-plugin-querystring/src/flow-nodes.yml:139]: newline

File: flow-nodes/api-builder-plugin-querystring/src/flow-nodes.yml [hunk 6]
```diff
@@ -5,123 +5,135 @@ flow-nodes:
...
         parameters:
           str:
+            name: URL query string
             description: The URL query string to encode.
             required: true
             initialType: string
             schema:
               type: string
           sep:
+            name: Substring sep
```
Comment (wickedest) [inline flow-nodes/api-builder-plugin-querystring/src/flow-nodes.yml:61]: "Substring sep" => "Key/value separator"

File: flow-nodes/api-builder-plugin-querystring/src/flow-nodes.yml [hunk 7]
```diff
@@ -5,123 +5,135 @@ flow-nodes:
...
             description: The substring used to delimit key and value pairs in the query string.
             required: false            
             initialType: string
             schema:
               type: string
               default: '&'
           eq:
+            name: Substring eq
```
Comment (wickedest) [inline flow-nodes/api-builder-plugin-querystring/src/flow-nodes.yml:69]: "Substring eq" => "Key/value separator"

**Pull request merged**